web-app/spec/workers/read_api_hazards_spec.rb

```diff
@@ -0,0 +1,99 @@
+    let(:json_response) {'{
+      "drawingInfo": {
+        "renderer": {
+          "uniqueValueInfos": [
+            {
+              "symbol": {},
+              "value": "High Winds",
+              "label": "High Winds",
+              "description": ""
+            },
+            {
+              "symbol": {},
+              "value": "Much Above Normal Temperatures",
+              "label": "Much Above Normal Temperatures",
+              "description": ""
+            },
+            {
+              "symbol": {},
+              "value": "Much Below Normal Temperatures",
+              "label": "Much Below Normal Temperatures",
+              "description": ""
+            },
+            {
+              "symbol": {},
+              "value": "Significant Waves",
+              "label": "Significant Waves",
+              "description": ""
+            },
+            {
+              "symbol": {},
+              "value": "Enhanced Wildfire Risk",
+              "label": "Enhanced Wildfire Risk",
+              "description": ""
+            }
+          ]
+        },
+        "transparency": 0,
+        "labelingInfo": []
+      }
+    }'}
+
+    let(:json_features_response){'{
+      "features": [
+        {
+          "attributes": {
+            "start_date": 1488412800000,
+            "end_date": 1488758400000
+          },
+          "geometry": {
+            "rings": [
+              [
+                [
+                  -140.9989123280414,
+                  61.894541385476657
+                ],
+                [
+                  -140.99738532777064,
+                  60.306829383369461
+                ],
+                [
+                  -140.99715632800201,
+                  60.306791383415657
+                ],
+                [
+                  -140.52319032717179,
+                  60.221865383580621
+                ],
+                [
+                  -140.45263832739681,
+                  60.309367383498056
+                ]
+              ]
+            ]
+          }
+        }
+      ]
+    }'}
+    
+    it "read apis hazards" do
+      allow(TestApiWeatherHazards).to receive(:get_json_from_url).with(url) {json_response}
+      allow(TestApiWeatherHazards).to receive(:get_hazard_keys).with(json_response) {current_value}
+      allow(TestApiWeatherHazards).to receive(:get_json_from_type).with(url_features, "weather") {json_features_response}
+      allow(TestApiWeatherHazards).to receive(:parse_json_value).with(json_features_response, value_test, "weather") {true}
+      worker.perform(url, "weather")
```

Comment:
Same here. You should capture the results of `worker.perform()` and check their values with `expect` commands.